src/Ng/Ng.csproj

```diff
@@ -157,7 +157,10 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="Catalog2Dnx.cs" />
+    <Compile Include="Catalog2ElfieOptions.cs" />
```

Comment:
So there is no reason down the line we can create a new project with just this new files and what else needed right ?